### PR TITLE
Spanish version to 01 Styles / 02 Import Bootstrap

### PR DESCRIPTION
**Pull request by crsanti**
Added README_es.md.
Corrected some names.
Corrected quotation marks and indentation.

### CODE REVIEW

File: 01 Styles/02 Import Bootstrap/README.md [hunk 1]
```diff
@@ -64,20 +64,20 @@ used by Bootstrap, we need to set a font appropiated loader:
...
 public url.
 
 ```
 npm install file-loader --save-dev
 ```
 
-- Producing all the files (images, fonts) in separate files is something we are used to, but what happens if we have a lot small files like this? Initial load of the page would be slower requesting a big bunch of small files, another option that we have is to use [url-loader](https://github.com/webpack/url-loader), in this case the files is encoded as data inside the HTML... and we can combine file-loader / url-loader by adding a condition, encode only files that are smaller that a given size.
+- Producing all the files (images, fonts) in separate files is something we are used to, but what happens if we have a lot small files like this? Initial load of the page would be slower requesting a big bunch of small files, another option that we have is to use [url-loader](https://github.com/webpack/url-loader), in this case the files is encoded as data inside the HTML... and we can combine *file-loader* / *url-loader* by adding a condition, encode only files that are smaller that a given size.
```
Comment (RosanaRufer) [inline 01 Styles/02 Import Bootstrap/README.md]: a lot **of** small

Comment (RosanaRufer) [reply 01 Styles/02 Import Bootstrap/README.md]: like these (plural) ??

Comment (brauliodiez) [reply 01 Styles/02 Import Bootstrap/README.md]: Good catch @Totemika

Comment (crsanti) [reply 01 Styles/02 Import Bootstrap/README.md]: I'm not sure if _like this_ is pointing to file or file**s** (maybe _this way_?). I'll suggest remove it. Any suggestions? :)
I will change these changes later. Thanks @Totemika!.

**Pull request merged**